### DISCUSSION TITLE
can´t open wallet due to problem while syncing

### DISCUSSION THREAD

**Discussion by Younameit6605** [iotaledger/firefly#7913] [Help & Support]
![firefly error](https://github.com/iotaledger/firefly/assets/83139802/3f4d7bf7-4162-45a3-8279-2e1022b79b7b)

**Comment by begonaalvarezd** [comment]
Tanglebay changed the nodes, now it is https://iota-node.tanglebay.com try to login and update the nodes within the settings or you can just create a new profile restoring, the new profile will have updated nodes
